modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-ktor/infrastructure/ApiClient.kt.mustache

```diff
@@ -148,18 +182,13 @@ import {{packageName}}.auth.*
         return request(requestConfig, FormDataContent(body ?: Parameters.Empty), authNames)
     }
 
-    protected suspend fun <T: Any?> jsonRequest(requestConfig: RequestConfig<T>, body: Any? = null, authNames: kotlin.collections.List<String>): HttpResponse {
-        val contentType = (requestConfig.headers[HttpHeaders.ContentType]?.let { ContentType.parse(it) }
-                ?: ContentType.Application.Json)
-        return if (body != null) request(requestConfig, serializer.write(body, contentType), authNames)
-        else request(requestConfig, authNames = authNames)
-    }
```

Comment:
@Marek00Malik This code is not necessary anymore?

Reply:
Yes, the client is now serializing the object, it's not waiting for a serialized string payload.